MembershipFailureTest.slave_splits_and_eventually_merges_back

```
java.lang.AssertionError: expected:<6> but was:<5>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.internal.cluster.impl.MembershipUpdateTest.assertMemberViewsAreSame(MembershipUpdateTest.java:754)
	at com.hazelcast.internal.cluster.impl.MembershipFailureTest.slave_splits_and_eventually_merges_back(MembershipFailureTest.java:510)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK8/com.hazelcast$hazelcast/1455/testReport/junit/com.hazelcast.internal.cluster.impl/MembershipFailureTest/slave_splits_and_eventually_merges_back_fd_phi_accrual_/

--- a/hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipFailureTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipFailureTest.java
@@ -507,8 +507,14 @@ public class MembershipFailureTest extends HazelcastTestSupport {
         assertClusterSizeEventually(1, member3);
 
         assertOpenEventually(mergeLatch);
-        assertMemberViewsAreSame(getMemberMap(member1), getMemberMap(member2));
         assertMemberViewsAreSame(getMemberMap(member1), getMemberMap(member3));
+
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() {
+                assertMemberViewsAreSame(getMemberMap(member1), getMemberMap(member2));
+            }
+        });
     }
 
     @Test

